sdk/src/test/java/com/mapbox/maps/ViewAnnotationManagerTest.kt

```diff
@@ -326,8 +331,88 @@ class ViewAnnotationManagerTest {
     assertEquals(ViewAnnotationUpdateMode.MAP_FIXED_DELAY, viewAnnotationManager.getViewAnnotationUpdateMode())
   }
 
+  @Test
+  fun getViewAnnotationsSize() {
+    val viewAnnotationOptions = viewAnnotationOptions {
+      geometry(DEFAULT_GEOMETRY)
+    }
+    viewAnnotationManager.addViewAnnotation(view, viewAnnotationOptions)
+
+    val anotherView = mockView()
+    viewAnnotationManager.addViewAnnotation(
+      anotherView,
+      viewAnnotationOptions {
+        geometry(Point.fromLngLat(90.0, 90.0))
+      }
+    )
+    every { anotherView.removeOnAttachStateChangeListener(any()) } just Runs
+    assertEquals(2, viewAnnotationManager.annotations.size)
+  }
+
+  @Test
+  fun getViewAnnotationOptionsFromAnnotationsMap() {
```

Comment:
maybe add test for:
- checking adding same annotations? size should be const
- adding several times different annotations and between it check list size